feat: add cluster.hocon backup info (#1922)
emqtt/emq-docs-cn: en_US/configuration/configuration.md

:::

> Since version 5.1, when cluster config changes, EMQX makes a backup for `cluster.hocon` file before overwriting it.

Reviewer: ```suggestion
Since version 5.1, when cluster configuration changes, EMQX backups the `cluster.hocon` file before overwriting it.
```

The backup files are suffixed with a timestamp of the node's local time.

> There are at most 10 backup files kept.

Reviewer: ```suggestion
At most 10 backup files can be kept.
```
